polylint.js

```diff
@@ -28,16 +28,20 @@ var polylint = function polylint(path, options) {
   options.filter = function(){
     return false;
   };
-  options.redirect = "bower_components";
```

Comment:
This was undoing https://github.com/PolymerLabs/polylint/blob/master/bin/polylint.js#L175